pallets/runtime/tests/src/sto_test.rs

```diff
@@ -49,11 +49,35 @@ fn zero_price_sto_ext() {
 }
 
 #[test]
-fn invalid_fundraiser_ext() {
+fn fundraiser_expired_ext() {
     ExtBuilder::default()
         .cdd_providers(vec![AccountKeyring::Eve.public()])
         .build()
-        .execute_with(invalid_fundraiser);
+        .execute_with(fundraiser_expired);
+}
+
+#[test]
+fn modifying_fundraiser_window_ext() {
+    ExtBuilder::default()
+        .cdd_providers(vec![AccountKeyring::Eve.public()])
+        .build()
+        .execute_with(modifying_fundraiser_window);
+}
+
+#[test]
+fn freeze_unfreeze_fundraiser_ext() {
+    ExtBuilder::default()
+        .cdd_providers(vec![AccountKeyring::Eve.public()])
+        .build()
+        .execute_with(freeze_unfreeze_fundraiser);
+}
+
+#[test]
+fn stop_fundraiser_ext() {
+    ExtBuilder::default()
+        .cdd_providers(vec![AccountKeyring::Eve.public()])
+        .build()
+        .execute_with(stop_fundraiser);
```

Comment:
Let's introduce a higher order function `#[track_caller] fn test(logic: impl FnOnce())` and refactor the functions above with that.